doc/apps/streamlit.md

```diff
@@ -125,6 +125,34 @@ Auth0 configuration parameters. Check out the [sample app.](https://github.com/p
 
 ![auth0-login](../static/password/auth0-login.png)
 
+## Connecting to a DB
+
+You can use the [st.connection](https://docs.streamlit.io/develop/api-reference/connections/st.connection) API for connecting to a database from a Streamlit app.
+Check out this [sample app](https://github.com/ploomber/doc/tree/main/examples/streamlit/db-connection) that demonstrates DB connection.
+
+You can set the SQLAlchemy URI of your DB as an environment variable `DB_URI` and pass it using the `url` argument:
+
+```python
+import streamlit as st
+from os import environ
+
+conn = st.connection(name="sqlite",
```

Comment:
let's change `name` to a be a more generic one, also, mention that the value doesn't matter (I couldn't find any references but that's my guess since we're passing URL)

Reply:
[fixed](https://github.com/ploomber/doc/pull/228/commits/9205f768d6db6e333938fc8470569dcb7f4cafbf)